### PR TITLE
Adding details to the entity type definitions

### PR DESCRIPTION
**Pull request by jfjoly**

### CODE REVIEW

File: TrackOverview.md [hunk 1]
```diff
@@ -48,16 +48,18 @@ More info can be found below on how to get started.
  
 ### Entity Types
 **What?**
-* TBD
+* Entity type definitions are a mapping between the telemetry ingested to New Relic and the entities (objects) that users can interact with in New Relic UI. If you have telemetry from any source that is not supported out of the box in New Relic, you can propose a mapping to light up the New Relic experience.
```
Comment (jpvajda) [inline TrackOverview.md:51]:
```suggestion
* Entity type definitions are a mapping between the telemetry ingested to New Relic and the entities (objects) that users can interact with in New Relic UI. If you have telemetry from any source that is not supported out of the box in New Relic, you can propose a mapping to ingest data into the New Relic experience.  
```

Comment (jpvajda) [reply TrackOverview.md:51]: the term "light up" may not be clear to all users, so I'd suggest just expressing what the user can do with entity type definitions.

**Review by jpvajda — APPROVED** [review]
I had one suggestion that is optional. 😄

File: TrackOverview.md (hunk 1 above)
Comment (jfjoly) [reply TrackOverview.md:51]: the mapping enables the experiences for this entity type: Navigator, entities list view, entity overview, Lookout... I'm not sure of a better wording.

Comment (jpvajda) [reply TrackOverview.md:51]: Oh I see. so how about combining the language? to `ingest data into and light up` this way it's clear that the data ingest enables the UI to light up. 🤔

Comment (jfjoly) [reply TrackOverview.md:51]: I would prefer to keep the original wording.

Comment (jpvajda) [reply TrackOverview.md:51]: 👍

**Pull request merged**